tests/acceptance/stepDefinitions/middlewareContext.js

```diff
@@ -63,16 +63,19 @@ After(function () {
   })
 })
 
-Given(/^((?:(?!these|following).)*)(in the server|on remote server)(.*)$/, (st1, st2, st3) => {
+Given(/^((?:(?!these|following).)*\S)\s(in the server|on remote server)(.*)$/, (st1, st2, st3) => {
```

Comment:
`\S` matches non-white-space.
`\s` matches white-space.
The regex matches the command text up to the end of some word, then a single space, then "in the server" or "on remote server". If there are 2 spaces before "in"/"on" then it will fail an CI will fail. 
So we will know if excess white-space is accidentally added.